internal_worker.go

```diff
@@ -106,8 +107,11 @@ type (
 		// Defines how many concurrent executions for task list by this worker.
 		ConcurrentActivityExecutionSize int
 
-		// Defines rate limiting on number of activity tasks that can be executed per second.
-		MaxActivityExecutionsPerSecond float32
+		// Defines rate limiting on number of activity tasks that can be executed per MaxActivityExecutionRateRefreshDuration.
+		MaxActivityExecutionRate int
```

Comment:
For decisions we should figure out a high number that we manage by ourself, which might not be same as this and let users worry about only activity rate.